packages/lodestar-cli/src/cmds/account/cmds/validator/deposit.ts

```diff
@@ -7,7 +7,7 @@ const deprecatedDescription =
 - eth2.0-deposit-cli: https://github.com/ethereum/eth2.0-deposit-cli \
 - Ethereum Foundation launchpad: https://medalla.launchpad.ethereum.org";
 
-export const deposit: ICliCommand<{}, IAccountValidatorArgs & IGlobalArgs> = {
+export const deposit: ICliCommand<Record<string, unknown>, IAccountValidatorArgs & IGlobalArgs> = {
```

Comment:
Is the empty object type `{}` forbidden in this version? I'm afraid `Record<string, unknown>` might allow arbitrary keys.

Reply:
{} apparently accepts any non null type. As claimed by eslint rule

Reply:
I see it. For empty object you should use `Record<string, never>`

Reply:
That rule is lying, behaviour is not equivalent. I want that access of non declared keys to be wrong:
![Screenshot from 2020-12-02 19-17-32](https://user-images.githubusercontent.com/35266934/100920464-12051980-34d3-11eb-8b0d-8ad332b9ee3f.png)

Reply:
> That rule is lying, behaviour is not equivalent. I want that access of non declared keys to be wrong:
> ![Screenshot from 2020-12-02 19-17-32](https://user-images.githubusercontent.com/35266934/100920464-12051980-34d3-11eb-8b0d-8ad332b9ee3f.png)
yes, but you can call function fn1(1) and it would be ok.

![image](https://user-images.githubusercontent.com/8836210/100921099-619c1300-34dc-11eb-9ff0-08222fd42c76.png)

Eslint suggestion is to use Record<string, never>
![image](https://user-images.githubusercontent.com/8836210/100921418-d8391080-34dc-11eb-97ef-564808b904e9.png)

That said, that is out of scope for this PR as types were wrong in the first place

Reply:
`Record<never, never>` works great! Thanks for updating the PR